File: 본문/c/Code_2_03_1.c
#include <stdio.h>

int cnt = 1000;

int func1() {
	return 2021;
}

int func2(int pos) {
	cnt += 1;
	return cnt + pos;
}

int main() {
	printf("%d\n", func1()); // "2021"이라고 출력
	printf("%d\n", func2(500)); // "1501"이라고 출력
	printf("%d\n", func2(500)); // "1502"라고 출력
	return 0;
}

File: 본문/python/Code_2_03_1.py
cnt = 1000

def func1():
	return 2021

def func2(pos):
	global cnt # 전역 변수 cnt를 함수 내부에서 사용
	cnt += 1
	return cnt + pos

print(func1()) # "2021"이라고 출력
print(func2(500)) # "1501"이라고 출력
print(func2(500)) # "1502"라고 출력
